tests/lib/rules/no-arrow-function-computed-properties.js

```diff
@@ -19,20 +19,32 @@ const ruleTester = new RuleTester({
 });
 ruleTester.run('no-arrow-function-computed-properties', rule, {
   valid: [
-    'computed()',
-    'computed(function() { return 123; })',
-    "computed('prop', function() { return this.prop; })",
-    "computed('prop', function() { return this.prop; }).volatile()",
-    "computed.map('products', function(product) { return someFunction(product); })",
-    'other(() => {})',
-    'other.computed(() => {})',
+    {
+      code: `
+        computed('prop', function() { return this.prop; });
+        computed('prop', function() { return this.prop; }).volatile();
+        computed(() => { return 123; });
+        computed(() => { return "string stuff"; });
+        computed(() => []);
+        computed.map('products', product => { return someFunction(product); });
+      `,
+      options: [{ onlyThisContexts: true }],
+    },
+    `
+    computed();
+    computed(function() { return 123; });
+    computed('prop', function() { return this.prop; });
+    computed('prop', function() { return this.prop; }).volatile();
+    computed.map('products', function(product) { return someFunction(product); });
+    other(() => {});
+    other.computed(() => {});
+    `,
```

Comment:
Each line here should be kept as an individual test case. That allows the test runner to be much clearer with its output about which specific test cases are passing and failing.

Reply:
Should the same be done with the scenarios where the option is true?

Reply:
Yep!

Reply:
done!